database/migrations/2025.03.14T00.00.00.list-fill-variant-from-isNumeric.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
Would this migration be executed? it's already in the past, no?

Reply:
The filename is just for our reference and doesn't affect execution. afaik they're execute at as long as strapi doesn't list it in their internal migration table (`strapi_migrations`)